src/pages/Landingpage.js

```diff
@@ -10,7 +10,7 @@ export default function LandingPage() {
       <h1>
         <span className="sr-only">CATYUMMY</span>
         <img
-          src={headline_styled}
+          src={headlinestyled}
```

Comment:
When I do have it right in my mind, you could use camelCase over here 😊

Reply:
it was a try because of the problems with a svg